Makefile

```diff
@@ -142,7 +142,7 @@ extract: extract_$(VERSION)
 build: ##@ build game files
 build: build_$(VERSION)
 build_us: main dra weapon ric cen dre mad no3 np3 nz0 sel st0 wrp rwrp mar tt_000 tt_001
-build_hd: dra $(BUILD_DIR)/WRP.BIN tt_000
+build_hd: dra cen $(BUILD_DIR)/WRP.BIN tt_000
```

Comment:
I'm thinking part of your build issues may be that the only stage overlay we have here is wrp, done as `$(BUILD_DIR)/WRP.BIN`, while you add `cen`

Reply:
The issue appears to be missing tile and sprite entries in the splat config. It looks like I was getting those from a `us` extract. I'm going to pull this down while I work out those offsets.